app/errorHandlers.js

```diff
@@ -0,0 +1,15 @@
+exports.handle400s = (err, req, res, next) => {
+  console.log(err);
+  if (err.code === "22P02" || err.code === "23502") {
+    res.status(400).send({ msg: "Bad request" });
+  } else if (err.code === "23503") {
+    res.status(404).send({ msg: "Not found" });
+  } else if (err.status === 400) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+};
```

Comment:
You shouldn't push code that doesn't affect the ticket you are on - you have no errors yet.

Reply:
You are totally right, will keep them out of the code for now :)